conf/messages.en

```diff
@@ -186,36 +186,44 @@ fosterCarer.error.required = Select yes if fosterCarer
-highIncomeChildBenefitCharge.error.required = Select highIncomeChildBenefitCharge
-highIncomeChildBenefitCharge.agent.error.required = Select highIncomeChildBenefitCharge
-highIncomeChildBenefitCharge.change.hidden = HighIncomeChildBenefitCharge
+################################### CHILD BENEFIT INCOME ###########################################################################################
+childBenefitIncome.title = Was your income over £50,000?
+childBenefitIncome.agent.title = Was your client's income over £50,000?
+childBenefitIncome.error.required = Select yes if your income was over £50,000
+childBenefitIncome.agent.error.required = Select yes if your client's income was over £50,000
+
+################################### HIGH INCOME CHILD BENEFIT CHARGE ###########################################################################################
+
+highIncomeChildBenefitCharge.title = High Income Child Benefit Charge
+highIncomeChildBenefitCharge.agent.title = High Income Child Benefit Charge
+highIncomeChildBenefitCharge.details.residence.summary = Understanding High Income Child Benefit Charge
+highIncomeChildBenefitCharge.details.residence.p2 = You may have to pay the High Income Child Benefit Charge if you or your partner have an individual income that’s over £50,000 and either:
+highIncomeChildBenefitCharge.details.agent.p2 = Your client may have to pay the High Income Child Benefit Charge if they or their partner have an individual income that’s over £50,000 and either:
+highIncomeChildBenefitCharge.details.residence.p3 = It does not matter if the child living with you is not your own child.
+highIncomeChildBenefitCharge.details.agent.p3 = It does not matter if the child living with your client is not their own child.
+highIncomeChildBenefitCharge.details.bullet1 = you or your partner get Child Benefit
+highIncomeChildBenefitCharge.details.agent.bullet1 = they or their partner get Child Benefit
+highIncomeChildBenefitCharge.details.bullet2 = someone else gets Child Benefit for a child living with you and they contribute ar least an equal amount towards the child's upkeep
+highIncomeChildBenefitCharge.details.agent.bullet2 = someone else gets Child Benefit for a child living with them and they contribute ar least an equal amount towards the child's upkeep
+highIncomeChildBenefitCharge.p1 = The partner with the higher income may have to pay High Income Child Benefit Charge.
+highIncomeChildBenefitCharge.agent.p1 = The partner with the higher income may have to pay High Income Child Benefit Charge.
+highIncomeChildBenefitCharge.heading = Whose income was higher?
+highIncomeChildBenefitCharge.agent.heading = Whose income was higher?
+highIncomeChildBenefitCharge.option1 = My income
+highIncomeChildBenefitCharge.option2 = My partner’s income
+highIncomeChildBenefitCharge.option3 = I do not have a partner
+highIncomeChildBenefitCharge.agent.option1 = My client's income
```

Comment:
Please change the type of apostrophe to " ’ " rather than " ' ".  The latter type get removed from messages.